CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Replaced `DAPP_DEPLOY_BLOCK_HASH` with `DAPP_DEPLOYMENT_BLOCK_NUMBER`.
 
+### Fixed
+
+- Fixed a problem where `redis` and `server-manager` did not receive all environment variables
+
```

Comment:
We should remove this entry because the go supervisor wasn't released in previous versions.